### DISCUSSION TITLE
please support unicode character

### DISCUSSION THREAD

**Discussion by sugizo** [niklasf/python-chess#995] [Ideas]
```
[Event "Budapest"]
[Site "Budapest"]
[Date "1911.??.??"]
[White "Lasker, Emanuel"]
[Black "Breyer, Gyula"]
[Result "0-1"]
[ECO "C21"]

1. e4 e5 2. d4 exd4 3. c3 d5 4. exd5 ♕xd5 5. cxd4 ♘f6 6. ♘f3 ♗b4+ 7. ♘c3 O-O 8. ♗e2 ♘e4 9. ♗d2 ♗xc3 10. bxc3 ♘c6 11. O-O ♕a5 12. ♕c2 ♗f5 13. ♗d3 ♖fe8 14. ♗e1 ♕d5 15. c4 ♕d6 16. d5 ♘e5 17. ♗xe4 ♘xf3+ 18. gxf3 ♕g6+ 19. ♔h1 ♖xe4 20. ♕c3 ♖h4 21. ♖g1 ♖xh2+ 22. ♔xh2 ♕h5+ 23. ♔g3 ♕g5+ 24. ♔h2 ♕h4+ 25. ♔g2 ♕h3# 0-1
```

the chess pieces in unicode character, it's not work e.g. ♕, ♗

**result** 
only pawn move on pgn above is created

**expected result** 
all pieces can work with unicode character

thanks n best regards

**Comment by niklasf** [comment]
Mhh ... Unicode characters like that are not valid in PGN. Which software is creating this output?

**Reply by sugizo** [reply]
scid vs mac
but 
found in pgn files like this on some websites
